test/mitmproxy/tools/console/test_master.py

```diff
@@ -0,0 +1,10 @@
+import os
+import tempfile
+
+from mitmproxy.tools.console import master
+
+
+def test_try_unlink_file_not_exists():
```

Comment:
Can we use https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html please? Cleaning up after mkdtemp is a bit annoying.